### DISCUSSION TITLE
workspaces wrap/cycle

### DISCUSSION THREAD

**Discussion by zoovasoup** [hyprwm/Hyprland#5924] [General]
is there a way to disable workspaces wrap/cycle to the first or last active workspace?

**Comment by Sebbito** [comment]
https://wiki.hyprland.org/Configuring/Dispatchers/#workspaces

I assume you have your keybinds for changing workspaces in the form of:

`bind = $mainMod SHIFT, l, workspace, e+1`

Just change the `e+1` to something like `r+1` and it shouldnt wrap.

**Reply by zoovasoup** [reply]
https://github.com/zoovasoup/nixos-config/blob/misterio-setup/home-manager/hyprland/default.nix#L254-L260

I already had both in my keybinding 

What i want is, when i press the `e+1` keybind , it will stop at the end of the active workspaces 

For example i have workspaces 1,3,6 active and im currently at workspaces 3, then when i press the `e+1` keybind i want it to stop at the workspace 6 and doesnt loop to workspace 1

**Reply by Sebbito** [reply]
Ah I'm sorry, now i understand. Sadly I couldn't find anything like that now that I've looked at it again.

**Comment by code-IM-perfect** [comment]
Would like to see this implemented too

Relevant Issue: #6915 (it was force closed in the great purge)
